pkg/reconciler/v1alpha1/revision/resources/service_test.go

```diff
@@ -29,6 +29,16 @@ import (
 	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
 )
 
+func serviceLabels(labels map[string]string) map[string]string {
+	ret := map[string]string{
+		serving.ServiceTypeKey: serving.ServiceTypePublic,
+	}
+	for k, v := range labels {
+		ret[k] = v
+	}
+	return ret
+}
```

Comment:
I think @vagababov just added some functions that make most of the body of this go away

Reply:
+1
just use `pkg/resources/meta.go`